src/MomentoTaggedCache.php

```diff
@@ -27,21 +34,51 @@ public function put($key, $value, $ttl = null)
     public function get($key, $default = null): mixed
     {
         $tags = $this->tags->getNames();
+        if (!self::validateTags($tags)) {
+            return false;
```

Comment:
The Laravel docs don't discuss validation of tags (or throwing exceptions) at all. By my reading of the doc, we should handle this as a cache miss and return `null` instead.

Reply:
Since `MomentoTaggedCache` extends `TaggedCache`, we want to return `bool` for `get`.
But now I think about it, I should not decide this behavior of returning false in case `$tags` are empty.
`tags` method in `MomentoStore` should be able to throw an exception if users pass empty tags based on the source code (more so `func_get_args()` will thrown an error if an argument is empty):
```php
public function tags($names)
    {
        return new TaggedCache($this, new TagSet($this, is_array($names) ? $names : func_get_args()));
    }
```

Reply:
I wrote a test to put with an empty tag and it was actually success though we don't want encourage this behavior.
I agree that we should return `null` in case users pass an empty tag.

Reply:
But in case of put, `false` will be returned.

Reply:
`TaggedCache` doesn't have a `get` method to override. The `get` method here is implementing the definition in `CacheInterface::get`, which has a return value of `mixed`. That makes sense, because if we had to return a `bool` we could never return an actual cache value.

I think this works for now, but the more I think about it the more I think it might make sense to throw an exception here instead. Because the only thing that makes a tag technically invalid is it being empty, and I can imagine how it might be easy to accidentally pass empty tags, it seems like it might be better to alert the user of the error than to return the equivalent of a cache miss. I don't want to hang up the PR for this issue and I think this is good for now, but I'd like to discuss it a little more at some point.